src/validation/error.messages.ts

```diff
@@ -74,4 +74,9 @@ export enum ErrorMessages {
   ENTITY_NAME_INVALID_CHARACTERS = "The name of the overseas entity must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   LEGAL_FORM_INVALID_CHARACTERS = "Legal form must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
   LAW_GOVERNED_INVALID_CHARACTERS = "Governing law must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
+  REGISTER_NAME_INVALID_CHARACTERS = "Name of register must only include letters a to z, numbers, and special characters such as hyphens, spaces and apostrophes",
```

Comment:
Rather than add REGISTER_NAME_INVALID_CHARACTERS and REGISTRATION_NUMBER_INVALID_CHARACTERS messages, please use the existing PUBLIC_REGISTER_NAME_INVALID_CHARACTERS and PUBLIC_REGISTER_NUMBER_INVALID_CHARACTERS enums. Message texts are the same.